src/app/core/session/session-service/synced-session.service.spec.ts

```diff
@@ -316,56 +317,39 @@ describe("SyncedSessionService", () => {
       });
     });
 
-    it("behaves correctly when the sync fails and the local login succeeds", (done) => {
-      const localLogin = spyOn(localSession, "login").and.returnValue(
-        Promise.resolve(LoginState.LOGGED_IN)
-      );
-      const remoteLogin = spyOn(remoteSession, "login").and.returnValue(
-        Promise.resolve(ConnectionState.CONNECTED)
-      );
-      const syncSpy = spyOn(sessionService, "sync").and.returnValue(
-        Promise.reject()
-      );
+    it("behaves correctly when the sync fails and the local login succeeds", fakeAsync(() => {
```

Comment:
I changed the syntax of this and the following test because they initially failed due to some unhandled rejected promises.